torch/distributions/gumbel.py

```diff
@@ -45,5 +46,17 @@ def log_prob(self, value):
         z = (value - self.loc) / self.scale
         return -(self.scale.log() + z + torch.exp(-z))
 
+    @property
+    def mean(self):
+        return self.loc + self.scale * self._euler_constant
+
+    @property
+    def stddev(self):
+        return math.pi * self.scale / math.sqrt(6)
```

Comment:
I am not sure if I am right: `(math.pi / math.sqrt(6)) * self.scale` could possibly save a tensor op.